WordPress/Classes/Services/AccountService+Swift.swift

```diff
@@ -0,0 +1,82 @@
+import Foundation
+import WordPressShared
+import ShareExtensionCore
+
+extension AccountService {
+    func setupAppExtensions() {
+        let context = coreDataStack.mainContext
+        context.perform {
```

Comment:
I replaced `performAndWait` with `perform`. Making blocking calls is a big red flag.  Since there is already an existing `DispatchQueue.main.async` further in the method, it should not hurt.

Reply:
This is something for a followup, and my concurrency chops are poor given the little time I've been spending in the apps recently.

But reading "Since there is already an existing DispatchQueue.main.async further in the method, it should not hurt." makes me slightly uneasy because that's something we the coders need to know and remember. 

If I understand correctly, we could annotate with `@MainActor` to get the compiler to enforce this. Or am I missing something? That might be a bigger change than we have time for ATM, but we could log a [name of issue/task in Linear] for it?